lib/rspec-console/exceptions.rb

```diff
@@ -0,0 +1,8 @@
+module RSpecConsole
+  class VersionError < StandardError
+    'Please use RSpec 2.9.10 or later'
```

Comment:
Okay, so the code you wrote does not do what you think it does:

![irb2](https://cloud.githubusercontent.com/assets/297060/7779796/92675128-00a5-11e5-9ce1-7789e4462194.png)

The thrown exception does not come with the message "Please use..."